eth-contracts/contracts/DelegateManager.sol

```diff
@@ -190,6 +197,8 @@ contract DelegateManager is InitializableV2 {
     ) external returns (uint newDelegateAmount)
     {
         _requireIsInitialized();
+        _requireClaimsManagerAddressIsSet();
```

Comment:
should this be in _claimPending instead of the caller? i was looking for the usage of claimsManager address here and realized it is inside that helper

Reply:
same thing applies to undelegateStake

Reply:
and in claimRewards, the claimsManagerAddress is used by _validateClaimRewards only

Reply:
per offline discussion, going to table this